remoteFortisoar/info.json

```diff
@@ -0,0 +1,242 @@
+                    "type": "select",
+                    "visible": true,
+                    "editable": true,
+                    "required": true,
+                    "options": [
+                        "GET",
+                        "POST",
+                        "PUT",
+                        "DELETE"
+                    ],
+                    "onchange": {
+                        "GET":[
+                            {
+                                "title": "Query params",
+                                "name": "params",
+                                "description": "Any query params to send as part of the url",
+                                "type": "text",
+                                "visible": true,
+                                "editable": true,
+                                "required": false,
+                                "placeholder": "{'id':'1'}"
+                            },
+                            {
+                                "title": "Headers",
+                                "name": "headers",
+                                "description": "An dict full of headers to add to the request",
+                                "type": "text",
+                                "visible": true,
+                                "editable": true,
+                                "required": false
+                            }
+                        ],
+                        "POST":[
+                            {
+                                "title": "Query params",
+                                "name": "params",
+                                "description": "Any query params to send as part of the url",
+                                "type": "text",
+                                "visible": true,
+                                "editable": true,
+                                "required": false,
+                                "placeholder": "{'id':'1'}"
+                            },
+                            {
+                                "title": "Body",
+                                "name": "body",
+                                "description": "The body (which won't be encoded) to send with the request",
+                                "type": "text",
+                                "visible": true,
+                                "editable": true,
+                                "required": false,
+                                "tooltip": "Body for POST OR PUT methods"
+                            },
+                            {
+                                "title": "Headers",
+                                "name": "headers",
+                                "description": "An dict full of headers to add to the request",
+                                "type": "text",
+                                "visible": true,
+                                "editable": true,
+                                "required": false
```

Comment:
a default header of accept: Application/json would be nice to have

Reply:
Done